yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -403,10 +393,37 @@ func reportStreamingProgress() {
 	}
 }
 
+func calculateStreamingProgress() {
+	var err error
+	for {
+		totalEventCount, totalEventCountRun, err = metaDB.GetTotalExportedEventsByExporterRole(exporterRole, runId)
+		if err != nil {
+			utils.ErrExit("failed to get total exported count from metadb: %w", err)
+		}
+
+		throughputInLast3Min, err = metaDB.GetExportedEventsRateInLastNMinutes(runId, 3)
+		if err != nil {
+			utils.ErrExit("failed to get export rate from metadb: %w", err)
+		}
+		throughputInLast10Min, err = metaDB.GetExportedEventsRateInLastNMinutes(runId, 10)
+		if err != nil {
+			utils.ErrExit("failed to get export rate from metadb: %w", err)
+		}
+		if disablePb && callhome.SendDiagnostics {
+			// to not do unneccessary frequent calls to metadb in case we only require this info for callhome
+			time.Sleep(12 * time.Minute)
```

Comment:
is the callhome frequency also the same?

Reply:
its 15 mins, so kept this as 12 mins